rocketpy/sensors/sensors.py

```diff
@@ -8,6 +9,265 @@
 class Sensors(ABC):
     """Abstract class for sensors
```

Comment:
class Environment
class Motor
class NoseCone
class Tail
class Parachute
class Rocket
class Flight

They are all singular

```suggestion
class Sensor(ABC):
    """Abstract class for sensors
```

Reply:
Agreed, this is an important change, don't know how I didn't see that in the other sensors PR. Great comment.